LondonDataServices.IDecide.Manage.Server.Tests.Acceptance/Brokers/TestWebApplicationFactory.cs

```diff
@@ -30,10 +27,6 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
             builder.ConfigureServices((context, services) =>
             {
                 OverrideSecurityForTesting(services);
```

Comment:
Can you amend the config section above to be the same as the other test project?  Also add the appsettings.json and appsettings.Acceptance.json and the git ignore file for that please.

```cs
            builder.ConfigureAppConfiguration((context, config) =>
            {
                var testProjectPath = Path.GetFullPath(
                    Path.Combine(AppContext.BaseDirectory, "..", "..", ".."));

                config
                    .AddJsonFile("appsettings.json", optional: true)
                    .AddJsonFile(Path.Combine(testProjectPath, "appsettings.json"), optional: true)
                    .AddJsonFile(Path.Combine(testProjectPath, "appsettings.Acceptance.json"), optional: true)
                    .AddEnvironmentVariables();
            });
```